mail/dovecot2/Portfile

```diff
@@ -105,150 +132,95 @@ variant libstemmer description {Use libstemmer for full-text search} {
     configure.args-append       --with-libstemmer
 }
 
+variant postgresql84 \
+    conflicts postgresql96 postgresql10 postgresql11 postgresql12 \
+    description "Enable PostgreSQL 8.4 support" {
+    depends_lib-append          port:postgresql84
+    configure.env-append        PG_CONFIG=${prefix}/lib/postgresql84/bin/pg_config
+    configure.args-append       --with-pgsql
+}
+
 variant postgresql96 \
-    conflicts postgresql82 postgresql83 postgresql84 \
-        postgresql90 postgresql91 postgresql92 postgresql93 postgresql94 \
-        postgresql95 \
+    conflicts postgresql84  postgresql10 postgresql11 postgresql12 \
     description "Enable PostgreSQL 9.6 support" {
     depends_lib-append          port:postgresql96
     configure.env-append        PG_CONFIG=${prefix}/lib/postgresql96/bin/pg_config
     configure.args-append       --with-pgsql
 }
 
-variant postgresql95 \
```

Comment:
If you're concerned with an automated upgrade path, you could obsolete the old variants in a way that pulls in a newer one. See for example the postgresql92 variant in postfix.

Reply:
what would you suggest that I did for regarding automated upgrade paths then ??